test-resources/diffs/2.diff

```diff
@@ -0,0 +1,43 @@
+diff --git a/src/formatting_stack/formatters/trivial_ns_duplicates.clj b/src/formatting_stack/formatters/trivial_ns_duplicates.clj
```

Comment:
Are you able to have this (& following) to be a bit smaller, link 1.diff?

Reply:
why?

Reply:
To make it more clear what a test makes (or brakes)

Reply:
The goal is to parse arbitrary diff files. I consider the *.diff files to be opaque / external input. Making them more readable is not really a concern. But we could discuss this tomorrow :).